File: inventario_frontend/src/models/vendor.ts
export class VendorForCreation {
  public name: string | undefined = undefined;
  public cuil?: string;
  public phone: string | undefined = undefined;
  public mail?: string;
  // public active = true;
  public description?: string;
}

export class Vendor extends VendorForCreation {
  public id!: string;
}

File: Inventario/DataAccess/Models/Vendor.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace Data
{
    public class Vendor: Entity
    {
        [Required]
        public string Name { get; set; }
        public string? CUIL { get; set; }
        public string Phone { get; set; }
        public string? Mail { get; set; }
        //public bool Active { get; set; }
        public string? Description { get; set; }
    }
}
